internal/handlers/common/aggregations/sort.go

```diff
@@ -54,6 +54,8 @@ func newSort(stage *types.Document) (Stage, error) {
 }
 
 // Process implements Stage interface.
+//
+// If sort path is invalid, it returns a possibly wrapped types.DocumentPathError.
```

Comment:
I think this is where we want to return `ErrPathContainsEmptyElement` error instead of `types.DocumentPathError`. This error is specific to `sort` stage as far as I can tell, and if we return `ErrPathContainsEmptyElement` here, we don't need to add the logic in both tigris and pg. What do you think? :hugs:

Reply:
I like the idea of returning `ErrPathContainsEmptyElement`, but I don't understand how it will help us not add the logic in Tigris and PG handlers as we need to return `CommandError` there. But it seems that I might miss something.

Reply:
Fixed